web/modules/custom/dpl_login/src/Adgangsplatformen/Config.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace Drupal\dpl_login\Adgangsplatformen;
+
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\dpl_login\Exception\MissingConfigurationException;
+use function Safe\sprintf as sprintf;
+
+/**
+ * Structured access to configuration for Adgangsplatformen.
+ *
+ * Adgangsplatformen is implemented as a plugin for the OpenID Connect Drupal
+ * module but we need access to the stored values elsewhere in the system.
+ * This class provides structured access to what would otherwise be an map
+ * of strings.
+ */
+class Config {
```

Comment:
Nice with an isolated "entity" for handling Adgangsplatformen config. 👍

Reply:
Consider making a unit test for the class

Reply:
Implemented in eb712d5.